packages/lit-dev-server/src/redirects.ts

```diff
@@ -35,8 +35,8 @@ export const pageRedirects = new Map([
   // Old release-notes redirects to new v2 location.
   ['/docs/releases/release-notes/1.3.0/',      '/docs/v2/releases/release-notes/1.3.0/'],
   ['/docs/releases/release-notes/1.2.0/',      '/docs/v2/releases/release-notes/1.2.0/'],
-  // Old Lit tutorial redirect to new tutorial catalog
-  ['/tutorial/', '/tutorials/'],
+  // Old Lit tutorial redirect to new learn catalog
+  ['/tutorial/', '/learn/'],
```

Comment:
should there also be one that redirects the plural to /learn?
```suggestion
  ['/tutorial/', '/learn/'],
  ['/tutorials/', '/learn/'],
```